mkdocs.yml

```diff
@@ -1,7 +1,7 @@
 # Project information
-site_name: Playing the Internship Game
-site_description: 'A tutorial to python'
-site_author: 'Evy Kassirer'
+site_name: A Guide To All Things Internships
+site_description: 'Internship Guide'
+site_author: 'Evy Kassirer and Ronak Shah'
```

Comment:
I don't think you should be a site author (I put several times the time into this as you), but you can put credit to yourself at the bottom of the readme like I showed above

You'll also be a contributor on GitHub and it'll show up on your GitHub profile